pkg/strategy/production.go

```diff
@@ -125,6 +124,9 @@ func newProductionStrategy(jaeger *v1.Jaeger, existingSecrets []corev1.Secret) S
 	if indexCleaner != nil {
 		c.cronJobs = append(c.cronJobs, *indexCleaner)
 	}
+	if len(esRollover) > 0 {
+		c.cronJobs = append(c.cronJobs, esRollover...)
```

Comment:
@objectiser to help you understand the code. 

The `esRollover` arrays changes made on line 115 are not reflected if the `esRollover` array has been added to `c.cronJobs` before the modification.

Reply:
Yes, sorry - only looked at the changes, not the full code.